test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -2180,10 +2180,10 @@ def test_cgroup_cpuset_memory_spread_page(self):
         """
         self.load_config(self.cfg5 % ('false', '', 'true', 'false',
                                       'false', self.swapctl))
-        self.server.expect(NODE, {'state': 'free'},
-                           id=self.nodes_list[0], interval=3, offset=10)
-        a = {'Resource_List.select': '1:ncpus=1:mem=100mb:host=%s' %
-             self.hosts_list[0]}
+        id = self.nodes_list[0]
+        self.server.expect(NODE, {'state': 'free'}, id, interval=3, offset=10)
```

Comment:
Notice that the previous version specified "id=" to eliminate issues with the parameter order. To retain this, you want to specify "id=nid" here.

Reply:
Look at the side-by-side diff to see what I mean.